src/server.js

```diff
@@ -21,7 +21,7 @@ var compiler = webpack({
 });
 
 var app = new WebpackDevServer(compiler, {
-  contentBase: 'lib/',
+  contentBase: 'src/',
```

Comment:
I just updated the build so it copies over the index.html file to lib/ so this can stay the same.